package.json

```diff
@@ -1,40 +1,56 @@
 {
-  "private": true,
-  "scripts": {
-    "test:compatibility": "./scripts/testCompatibility.sh"
-  },
-  "devDependencies": {
-    "@starptech/prettyhtml": "^0.10.0",
-    "cypress": "^5.3.0",
-    "diff": ">=3.5.0",
-    "husky": "^4.3.0",
-    "lerna": "^3.22.1",
-    "lerna-changelog": "^1.0.1",
-    "lodash": ">=4.17.13",
-    "lodash.template": ">=4.5.0",
-    "mixin-deep": ">=1.3.2",
-    "meow": "8.0.0",
-    "prettier": "^1.15.3",
-    "set-value": ">=2.0.1",
-    "sirv-cli": "^0.4.4"
-  },
-  "prettier": {
-    "singleQuote": true,
-    "semi": true
-  },
-  "changelog": {
-    "repo": "SAP/luigi",
-    "labels": {
-      "breaking": ":boom: Breaking Change",
-      "enhancement": ":rocket: Added",
-      "bug": ":bug: Fixed",
-      "internal": ":house: Internal"
-    },
-    "cacheDir": ".changelog"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "./scripts/hooks/apply-formatters.sh && ./scripts/hooks/remove-test-prefixes.sh && ./scripts/hooks/generate-docu.sh && ./scripts/hooks/prevent-illegal-characters.sh"
-    }
-  }
+   "private": true,
+   "scripts": {
+      "test:compatibility": "./scripts/testCompatibility.sh",
+      "code-quality": "node code_quality.js -- mode=pre_commit",
```

Comment:
I think it would be best to have some options here. So with eslint only /or prettier only. 
Initially in the pre-commit one I don't think we can have both. We could leave prettier only, since the eslint warnings are too much, we probably won't fix them.